css/style.css

```diff
@@ -1101,13 +1101,20 @@ consentNextButton:hover{
     font-size:18px;
 }
 
-.messagesHeaderFont{
+.messagesHeaderFont,
+.messagesSubHeader               {
     font-family: 'Montserrat', sans-serif;
     font-weight: bold;
-    font-size:23px;
     line-height:25px;
     color: #6B6B6B;
 }
+.messagesHeaderFont {
+    font-size: 1.438rem;
```

Comment:
Just curious: are 1.438rem and 1.4 rem visually different in this case?

Reply:
I wanted to make the font sizes for the header to be 23px and the sub header to be 21px but use rem units.